svm40/svm40.h

```diff
@@ -124,6 +124,32 @@ int16_t svm40_read_measured_values_as_integers(int16_t* voc_index,
                                                int16_t* relative_humidity,
                                                int16_t* temperature);
 
+/**
+ * Returns the new measurement results as integers with raw values added.
+ *
+ * The firmware updates the measurement values every second. Polling data
+ * faster will return the same values. The first measurement is available one
+ * second after the start measurement command is issued. Any readout prior to
+ * this will return zero initialized values.
+ *
+ * @note Only available in measurement mode.
+ * @param   voc_index           VOC algorithm output with a scaling value of 10.
+ * @param   relative_humidity   Compensated ambient humidity in %RH with a
+ *                              scaling factor of 100.
+ * @param   temperature         Compensated ambient temperature in degree
+ *                              celsius with a scaling factor of 200.
+ * @param   voc_ticks_raw       Raw VOC output ticks as read from the SGP
+ *                              sensor.
+ * @param   relative_humidity   Uncompensated raw humidity in % RH as read from
+ *                              the SHT40 with a scaling factor of 100.
+ * @param   temperature_raw     Uncompensated raw temperature in degrees celsius
+ *                              as read from the SHT40 with a scaling of 200.
+ * @return  NO_ERROR on success, an error code otherwise
+ */
+int16_t svm40_read_measured_values_as_integers_with_raw_parameters(
```

Comment:
```suggestion
int16_t svm40_read_measured_values_as_integers_with_raw_params(
```
slightly shorter.. I still think it's too long and I'm not too happy with the "raw params" part, but can't see anything better either :/